language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/BallerinaTextDocumentService.java

```diff
@@ -143,7 +143,7 @@ void setClientCapabilities(LSClientCapabilities clientCapabilities) {
                     position.getPosition());
             try {
                 return LangExtensionDelegator.instance()
-                        .completion(position, context, this.serverContext, cancelChecker);
+                        .completion(position, context, this.serverContext, null);
```

Comment:
@nadeeshaan can you please verify this change again? Looks incorrect.

Reply:
Rectified.